src/com/pentalog/pentastagiu/homework2/shop/Customer.java

```diff
@@ -0,0 +1,43 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+public class Customer {
+    private int idCustomer;
```

Comment:
pentru fielduri dintr-o clasa, e ok sa nu repetam numele clasei in numele fieldurilor: fieldul e private si cand il folosim aici e clar ca ne referim la id-ul unui customer; la fel si setteri/getteri generati, va fi clar ca lucram cu id-ul unui customer pentru ca vom folosi o referinta cu un nume sugestiv: firstCustomer.getId()/customer.getId()/...